README.md

```diff
@@ -48,6 +48,9 @@ etc...
 
 ## Writing DSLs
 
+### Example Job
+An example job can be found [here](../blob/master/platform/jobs/exampleJob.groovy). 
+
 ### Shared Constants:
 In order to make use of shared constants in a DSL script:
 * Have a try/catch to parse the Yaml file into a map
```

Comment:
@aphanse I know this was already in the code, but possibly change from "Have a try/catch" to "Use a try/catch"